priv/repo/migrations/20170419143747_create_entity_status.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CoursePlanner.Repo.Migrations.CreateEntityStatus do
+  use Ecto.Migration
+
+  def change do
+    execute("""
+          CREATE TYPE entity_status AS ENUM (
+            'Planned',
+            'Active',
+            'Finished',
+            'Graduated',
```

Comment:
I thought `Graduated` wouldn't be necessary because we could consider a `Finished` student as `Graduated`